box/boxnavigator.py

```diff
@@ -300,6 +339,9 @@ def __init__(
         distance_threshold: int,
         forward_increment: float,
         rotation_increment: float,
+        ahead_box=None,
+        chance_of_random_action: float = 0.25,
```

Comment:
Why does the perfect navigator take random actions?

Reply:
It does not. That was on accident.